### DISCUSSION TITLE
Update to Bropat v3.0

### DISCUSSION THREAD

**Discussion by davidkalifa** [fuatakgun/eufy_security#1113] [General]
Hi there,

Since Bropat updated to v3.0 with a plethora of new devices. What can be done on our end to implement this new version please?

Thanks a lot

**Comment by fuatakgun** [comment]
Get eufy security ws add-on with 1.8 version and test it

**Reply by davidkalifa** [reply]
Amazing, just as you replied here I installed the update! 
Screamed a loud: YEAH! when I saw my T8530 sensors. 

Thanks a lot. Going to test it out

**Reply by jamesdeck** [reply]
I'm trying to test a lock T8506 with the new version but it's still showing two debug items instead of the actual lock.  The person is working now however.   Maybe I'm doing something wrong?

<img width="1027" alt="screenshot" src="https://github.com/fuatakgun/eufy_security/assets/60999319/958c0388-b9ab-417c-aaa2-e107cc9dfc2c">

**Comment by thomasdg** [comment]
I did the update the second version 1.8.0 came out, but would love to be able to use the "open" command for my smartdrop which was introduced in the latest version. If I look in the developer menu of Home Assistant a lot of commands seem to be listed in the info about the binary_sensor by no idea for the moment on how to "call" those commands... The SmartDrop appears as "SmartDrop Debug (device)" or "binary_sensor.smartdrop_debug_device" with a whole list of attributes.

**Reply by jennateevin** [reply]
Ditto on this. I was just about to post about Smart Drop. Some way to tell when it is opened would be so useful too?

**Reply by thomasdg** [reply]
I have been trying to look around in the code of both bropat's part and this one, but have to admit I get lost trying to merely implement the "open" command :)

**Comment by nigelmwilliams7** [comment]
Hi all
I am investigating ways in which I could possibly integrate eufy security centre with Smartthings.
The Use Case is changing rhe security mode in Eufy inline with the home location mode in Smartthings.
Currently I am using Notifications from Smartthings to trigger macros in the Eufy Security app.

There must be a better way to do this as I can see you clever lot seem to be able to access the eufy infrastructure...
